packages/caravan-bitcoin/src/signatures.ts

```diff
@@ -117,3 +122,9 @@ function multisigSignatureBuffer(signature) {
   signatureBuffer.set(Buffer.from(s), 64 - s.byteLength);
   return signatureBuffer;
 }
+
+export const isValidSignature = (
```

Comment:
I extracted this so I can use it in @caravan/psbt code. 

One thing that's a little crazy is that I couldn't get ECPair to work with bitcoinjs-lib v6 because the ecc lib wouldn't validate, but it works in v5. So I keep the sig validation with the v5 version of ECPair and that get tx validation working.

Good news is that there might be a solution in the works: https://github.com/bitcoinjs/ecpair/issues/21

Reply:
This is being imported in psbt and now doesn't work for my dev environment. Tests fail when importing. See above for my comments on test failures.